### PR TITLE
Face Stepping Initialization

### PR DESCRIPTION
**Pull request by owenmylotte**
Speed and accuracy improvements for the radiation initialization.

### CODE REVIEW

File: src/radiation/radiation.cpp [hunk 1]
```diff
@@ -330,8 +374,11 @@ void ablate::radiation::Radiation::Initialize(solver::Range cellRangeIn) {
             stepcount++;
         }
     }
-    /** Cleanup*/
+    /** Cleanup */
     DMDestroy(&radsearch) >> checkError;
+    VecRestoreArrayRead(faceGeomVec, &faceGeomArray);
```
Comment (mmcgurn) [inline src/radiation/radiation.cpp:379]: Would you mind adding the error checking here?

File: src/radiation/volumeRadiation.cpp [hunk 2]
```diff
@@ -4,7 +4,9 @@
 ablate::radiation::VolumeRadiation::VolumeRadiation(const std::string& solverId1, const std::shared_ptr<domain::Region>& region, std::shared_ptr<domain::Region> fieldBoundary,
                                                     const PetscInt raynumber, const std::shared_ptr<parameters::Parameters>& options,
                                                     std::shared_ptr<eos::radiationProperties::RadiationModel> radiationModelIn, std::shared_ptr<ablate::monitors::logs::Log> log)
-    : Radiation(solverId1, region, fieldBoundary, raynumber, options, radiationModelIn, log), CellSolver(solverId1, region, options) {}
+    : Radiation(solverId1, region, fieldBoundary, raynumber, options, radiationModelIn, log),
+      CellSolver(solverId1, region, options),
+      solverRegionMinusGhost(std::make_shared<domain::Region>(solverId1 + "_minusGhost")) {}
```
Comment (mmcgurn) [inline src/radiation/volumeRadiation.cpp:9]: Is the solverRegionMinusGhost being used anywhere?

**Review by mmcgurn — CHANGES_REQUESTED** [review]
Looks good, just some small questions.

File: src/radiation/volumeRadiation.cpp (hunk 2 above)
Comment (owenmylotte) [reply src/radiation/volumeRadiation.cpp:9]: Ah, I had set the VolumeRadiation up to omit the ghost cells. During debugging I realized it made no difference so I removed it. I can get rid of that label.

**Review by mmcgurn — APPROVED** [review]

**Pull request merged**